ShopperInsights/src/main/java/com/braintreepayments/api/findeligiblepayments/FindEligiblePaymentMethodDetails.kt

```diff
@@ -0,0 +1,29 @@
+package com.braintreepayments.api.findeligiblepayments
```

Comment:
Can we move this out of the the `findeligiblepayments` directory? We use packge-private scope throughout the SDK, which relies on everything living in `com.braintreepayments.api`

Reply:
fixed